WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalContentEntity.kt

```diff
@@ -43,7 +43,7 @@ sealed class LocalContentEntityData {
 
     data class ReaderPostsData(val posts: ReaderPostList) : LocalContentEntityData()
     data class BloggingRemindersData(val reminders: List<BloggingRemindersModel>) : LocalContentEntityData()
-    data class SitesData(val sites: List<SiteModel>) : LocalContentEntityData()
+    data class SitesData(val sites: List<SiteModel>, val isLoggedOn: Boolean) : LocalContentEntityData()
```

Comment:
We should revisit in the future; especially if we drop the full-content migration and go with some sort of single-signon between the apps. 

FYI: I really like how the migration orchestrator works and the addition of `then`, `orElse` and `otherwise` feels very much like the `Specification` pattern. Not sure if that was intentional, but it reads as such. 👍

Reply:
> I really like how the migration orchestrator works and the addition of then, orElse and otherwise feels very much like the Specification pattern.

Thanks! The pattern was inspired by the concept of "[railway oriented programming](https://vimeo.com/113707214)", and is indeed like the specification pattern.